app/Models/Avatar.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const Axios = use('axios')
+const Logger = use('Logger')
```

Comment:
Hehe. `Axios` and `Logger` should be all lowercase, per JS convention.

Reply:
I'll fix that